sdk/eventhubs/microsoft-azure-eventhubs-extensions/pom.xml

```diff
@@ -17,7 +17,12 @@
   <version>3.3.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs-extensions;current} -->
 
   <name>Microsoft Azure SDK for Event Hubs Extensions</name>
-  <description>Extensions built on Microsoft Azure Event Hubs</description>
+  <description>Extensions built on Microsoft Azure Event Hubs${line.separator}
```

Comment:
What does the extensions lib contain? Do we have the equivalent of this in Track 2?

Reply:
No we don't. I just added it so it existed.